libnm-core/nm-utils.c

```diff
@@ -4448,11 +4448,10 @@ _nm_utils_inet6_is_token (const struct in6_addr *in6addr)
 gboolean
```

Comment:
Typos in the commit msg:
"Also, [is/it] is called from [...]"
[...]
"hug[e] settings, which could trick NetworkManage[r] to allocate [...]"